spec/requests/api/v1/recipe_search_spec.rb

```diff
@@ -30,4 +30,24 @@
       expect(response_body[:errors]).to eq("No results found")
     end
   end
+
+  # it "recent search doesn't show up" do 
+  #   get "/api/v1/recent_search"
```

Comment:
are these codes something we will utilize in the future? Otherwise, can we remove them and push the change?

Reply:
Good point, I removed them for now.  It runs into a similar issue as caching where my data persists on redis, I think circle CI version of redis dumps the data each time it is run or per test.